### PR TITLE
chore: cleanup some vanity npc logic

### PR DESCRIPTION
**Pull request by aronwk-aaron**
Don't setup talk for vanity npc's if they are no phrases
noGhosting ldfkey for vanity npc's

### CODE REVIEW

**Comment by EmosewaMC** [comment]
Please address the compiler warnings and errors

File: dGame/Entity.h [hunk 1]
```diff
@@ -85,6 +85,7 @@ class Entity {
 	bool GetPlayerReadyForUpdates() const { return m_PlayerIsReadyForUpdates; }
 
 	bool GetIsGhostingCandidate() const;
+	void SetIsGhostingCandidate(bool value) {m_IsGhostingCandidate = value;};
```
Comment (EmosewaMC) [inline dGame/Entity.h:88]: formatting

**Review by EmosewaMC — DISMISSED** [review]

**Review by Jettford — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Comment by Terrev** [comment]
i approve : )

youre welcome

**Review by Jettford — APPROVED** [review]

**Review by EmosewaMC — COMMENTED** [review]
Approved

**Pull request merged**